src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
make a global Random field and use it here

Reply:
[Random field declared](https://github.com/mate-academy/jv-lottery/pull/366/commits/a50c5743cb5b01fd1714c28099f99b455763f2e1)